drape_frontend/gps_track_point.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+namespace df
+{
+
+struct GpsTrackPoint
```

Comment:
I don't like idea of using GpsTrackPoint in 'df' because it makes GpsTrack classes depended of 'drape', that looks strange. Yes, 'maps' is not good place for GpsTrack because then there is cyclic dependence between 'map' and 'drape'. I think it would be better to move GpsTrack classes to the new project 'gps_tracking' and make drape depended of gps_tracking and map depended of drape and gps_tracking. 
Taking into consideration the time pressure we have now, I think we can temporary take it as you did.